src/game/scenes/melee.c

```diff
@@ -399,6 +400,19 @@ void handle_action(scene *scene, int player, int action) {
                         sd_pilot_set_player_color(player2->pilot, SECONDARY, p_a.colors[1]);
                         sd_pilot_set_player_color(player2->pilot, PRIMARY, p_a.colors[2]);
                     }
+                    // TODO in netplay, use the lobby names
+                    strncpy_or_truncate(player1->pilot->name, lang_get(player1->pilot->pilot_id + 20),
+                                        sizeof(player1->pilot->name));
+                    // 1player name strings end in a newline...
+                    if(player1->pilot->name[strlen(player1->pilot->name) - 1] == '\n') {
+                        player1->pilot->name[strlen(player1->pilot->name) - 1] = 0;
+                    }
+                    strncpy_or_truncate(player2->pilot->name, lang_get(player2->pilot->pilot_id + 20),
+                                        sizeof(player1->pilot->name));
+                    // 1player name strings end in a newline...
+                    if(player1->pilot->name[strlen(player1->pilot->name) - 1] == '\n') {
+                        player1->pilot->name[strlen(player1->pilot->name) - 1] = 0;
+                    }
```

Comment:
Feels this newline stripping here should be done elsewhere, but also player1 is done twice here.

Reply:
the lang strings don't have to end in a newline, but whatever TXT file the originals were compiled from all do.
i'd propose we keep the newline stripping, and add a `// TODO: lang: remove (the need for) newline stripping` comment or similar.

Reply:
Okay, @Nopey fixes the newline in another pull request, that's fine, but this code still is doing the operation twice to player1, and using `sizeof(player1->pilot->name)` to get the size of `player2->pilot->name`

Reply:
will fix